app/services/update_wikidata_stats.rb

```diff
@@ -1,32 +1,132 @@
 # frozen_string_literal: true
-
 require_dependency "#{Rails.root}/lib/wikidata_summary_parser"
 require_dependency "#{Rails.root}/lib/importers/wikidata_summary_importer"
+# require the installed wikidata-diff-analyzer gem
+require 'wikidata-diff-analyzer'
 
 class UpdateWikidataStats
+  STATS_CLASSIFICATION = [
+    'claims created', 'claims removed', 'claims changed',
```

Comment:
This will be easier to read if there's one entry per line (and you can also organize them into sections via comments). The closer this list matches up with the UI for how we display this data, the easier it will be to maintain both (as we potentially add new stats in the future).

Reply:
@ragesoss Currently the order of the entries is based on the structure of the output of wikidata-diff-analyzer because that helps while making the stats hash according to the same order. I have tried to add comments and make each entry in one line to make it easier to handle.